projects/Amlogic/kodi/appliance.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<settings version="1">
+
+  <section id="system">
+    <category id="display">
+      <group id="1">
+        <setting id="videoscreen.screenmode">
+          <visible>true</visible>
+          <default>0192001080060.00000pstd</default>
+        </setting>
+        <setting id="videoscreen.limitedrange">
+          <visible>false</visible>
+        </setting>
+        <setting id="videoscreen.limitguisize">
```

Comment:
This is based on kodi patches that aren't merged yet upstream and/or are probably included in a later commit.

Reply:
The same as https://github.com/LibreELEC/LibreELEC.tv/blob/b5026f9d283f716a3f562dfe419a0bf60f2bcafa/projects/Allwinner/kodi/appliance.xml#L7-L10

Reply:
which requires this patch https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/Allwinner/patches/kodi/0002-guisize.patch

Reply:
@Kwiboo did that PR go upstream? .. carrying a backported PR patch isn't (or shouldn't be) an issue and this one is needed in several places.

Reply:
@chewitt gui size is PR'ed but not merged

Reply:
Added Kodi patches from @chewitt in latest force-push to PR branch.